static/js/components/GroupSources.jsx

```diff
@@ -196,28 +217,56 @@ const GroupSources = ({ route }) => {
     );
   };
 
+  const renderAlias = (dataIndex) => {
+    const { id: objid, alias } = sources[dataIndex];
+
+    return (
+      <a href={`/source/${objid}`} key={`${objid}_alias`}>
+        {alias}
+      </a>
+    );
+  };
+
   // This is just passed to MUI datatables options -- not meant to be instantiated directly.
 
   const renderRA = (dataIndex) => {
     const source = sources[dataIndex];
-    return (
-      <div key={`${source.id}_ra`}>
-        {source.ra}
-        <br />
-        {ra_to_hours(source.ra)}
-      </div>
-    );
+    return <div key={`${source.id}_ra`}>{source.ra.toFixed(6)}</div>;
+  };
+
+  const renderRA_sex = (dataIndex) => {
+    const source = sources[dataIndex];
+    return <div key={`${source.id}_ra_sex`}>{ra_to_hours(source.ra)}</div>;
   };
 
   // This is just passed to MUI datatables options -- not meant to be instantiated directly.
   const renderDec = (dataIndex) => {
+    const source = sources[dataIndex];
+    return <div key={`${source.id}_dec`}>{source.dec.toFixed(6)}</div>;
+  };
+
+  const renderDec_sex = (dataIndex) => {
+    const source = sources[dataIndex];
+    return <div key={`${source.id}_dec_sex`}>{dec_to_hours(source.dec)}</div>;
+  };
+
+  const renderRedshift = (dataIndex) => {
     const source = sources[dataIndex];
     return (
-      <div key={`${source.id}_dec`}>
-        {source.dec}
-        <br />
-        {dec_to_hours(source.dec)}
-      </div>
+      <div key={`${source.id}_redshift`}>{source.redshift.toFixed(6)}</div>
```

Comment:
Dont use fixed point here - implies a redshfit precision which may not be achieved in reality. I.e., we dont want to render a redshift that is 0.06 as 0.060000